packages/@okta/vuepress-site/docs/guides/sign-in-with-facebook/configure-idp/index.md

```diff
@@ -4,13 +4,13 @@ title: Configure Facebook as an Identity Provider in Okta
 
 2.1. Sign in to your Okta org.
 
-2.2. On the main page, hover your cursor over **Users** until the menu opens, then click on **Social & Identity Providers**.
+2.2. If you use the Developer Console, hover your cursor over **Users** until the menu opens, and then click **Social & Identity Providers**. If you use the Classic UI, hover over **Security**, and then click **Identity Providers**.
```

Comment:
Pt 2.2 (for other providers) is missing from Facebook. I don't think that is intentional

Reply:
Oh, that's a good point. LinkedIn is missing it too.